egs2/ms_indic_18/asr1/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+lang=te # te ta gu
```

Comment:
Did you try the other languages?
It would be great if you also do so and upload a model and result later.

Reply:
I did not try the other languages yet. I will run the multilingual experiments next week (after interspeech) and release the best models.